File: HackerRank/Algorithms/2. Implementation/Grading_students.c
#include <stdio.h>

int main(){
    int n;
    scanf("%d",&n);
    for(int a0 = 0; a0 < n; a0++){
        int grade;
        scanf("%d",&grade);
        // your code goes here
        if ((grade >= 38) && (grade % 5) > 2) {
          grade = ((grade +2) / 5) * 5;
        }
        printf("%d\n", grade);
    }
    return 0;
}

File: HackerRank/Algorithms/2. Implementation/Grading_students.java
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        for(int a0 = 0; a0 < n; a0++){
            int grade = in.nextInt();
            // your code goes here
            if ((grade >= 38) && (grade % 5) > 2) {
              grade = ((grade + 2) / 5) * 5;
            }
            System.out.println(grade);
        }
    }
}
